### DISCUSSION TITLE
Help me!   I am stuck on the npm run login for custom clients of hubs cloud

### DISCUSSION THREAD

**Discussion by shinhochoi** [mozilla/hubs#4662] [Hubs Cloud Questions]
Hi everyone,
Please help me. I can not go further at  'npm run login ' described in the document https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html 

Previous my discussion titlel is  "Creating and Deploying Custom Clients : npm run login " . 

I guess many of users here already passed all the procedures of the document https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html. 

My hubs cloud env:
- AWS EC2 Entrerprise
- I followed the youtube : How to Deploy Hubs Cloud to AWS: Part 1 / part2 / part3. (ex, https://www.youtube.com/watch?v=2K7P8jFyHNc)
- Front domain : khub1.com
- Route 53 used

Question
- where can find backend host name for my hubs cloud in AWS dashboard and how does it  looks? 
 Beacause #npm run login ==> asks for host:  ==> what to enter for this?
  Even on the document, nothing mentioned about the host domain.  In the part2 
  some one say, I can find it in the route 53 dashboard.  I can not find it.  I can see only domain name of frontend such as
  khub1.com.    But it does not work for "npm run login".

  I want you to show some screen shot of route 53 or your work.

The document for Hubs client   make me crazy for several days.   

Hi, hubs cloud members,
Please be more  clear for  explaination.

**Comment by wmurphyrd** [comment]
Use your fontend domain in the `npm run login` command: khub1.com

**Comment by nupur711** [comment]
> Hi everyone, Please help me. I can not go further at 'npm run login ' described in the document https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html
> 
> Previous my discussion titlel is "Creating and Deploying Custom Clients : npm run login " .
> 
> I guess many of users here already passed all the procedures of the document https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html.
> 
> My hubs cloud env:
> 
> * AWS EC2 Entrerprise
> * I followed the youtube : How to Deploy Hubs Cloud to AWS: Part 1 / part2 / part3. (ex, https://www.youtube.com/watch?v=2K7P8jFyHNc)
> * Front domain : khub1.com
> * Route 53 used
> 
> Question
> 
> * where can find backend host name for my hubs cloud in AWS dashboard and how does it  looks?
>   Beacause #npm run login ==> asks for host:  ==> what to enter for this?
>   Even on the document, nothing mentioned about the host domain.  In the part2
>   some one say, I can find it in the route 53 dashboard.  I can not find it.  I can see only domain name of frontend such as
>   khub1.com.    But it does not work for "npm run login".
>   I want you to show some screen shot of route 53 or your work.
> 
> The document for Hubs client make me crazy for several days.
> 
> Hi, hubs cloud members, Please be more clear for explaination.

Did you find the solution for the same? I am stuck on the same
